modules/svg/rapid_features.js

```diff
@@ -16,7 +16,7 @@ let _actioned;
 export function svgRapidFeatures(projection, context, dispatch) {
   const RAPID_MAGENTA = '#da26d3';
   const throttledRedraw = _throttle(() => dispatch.call('change'), 1000);
-  const gpxInUrl = utilStringQs(window.location.hash).gpx;
+  const gpxInUrl = context.initialHashParams.hasOwnProperty('gpx');
```

Comment:
Was this a bug in how we were calculating the existence of a `gpx` param? Just trying to understand the reason for this change.

Reply:
A gotcha in the iD code is that there are a lot of places where different code will mess with `window.location.hash`.  So if other code is looking for props in there, it might not always find it.

Last year the `context.initialHashParams` was added to preserve it so that code anywhere can know whether iD was started with that hash param or not - it just feels safer to look there rather than look at the instantaneous hash value.